spec/unit/jobs/v3/delete_service_instance_job_spec.rb

```diff
@@ -46,7 +62,12 @@ module V3
 
             expect(subject.instance_variable_get(:@attempts)).to eq(1)
             expect(subject.instance_variable_get(:@first_time)).to eq(true)
-            expect(subject.instance_variable_get(:@request_failed)).to eq(true)
```

Comment:
I think I put in the original instance variable gets in this test, and it actually feels a bit dirty to reach into the object and break encapsulation.  I might have a think on how to fix the others as well (for instance, maybe attempts doesn't matter because we show that it fails after 3 tries).